arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
  allow_partial_joints_goal: True
gripper_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - gripper_joint1
    - gripper_joint2

  allow_partial_joints_goal: True
  gains:
    gripper_joint1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    gripper_joint2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1


      # Publish joint_states
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50